first_boot-displaymanager-20160805.json

```diff
@@ -0,0 +1,15 @@
+{
+  "properties": [],
+  "tags": [
+    "displaymanager"

```

Comment:
the tags don't relate to filenames. If searching for existing ones in the workspace of needles, better use `git grep <tag>`, e.g. `git grep displaymanager` which should show you the needle filenames mentioning this tag, e.g.

```
$ git grep displaymanager
change_password-displaymanager-20160520.json:    "displaymanager"
desktop-wait-to-login-after-virt-test-20160519.json:    "virttest-displaymanager"
displaymanager-20140621.json:    "displaymanager",
displaymanager-20160502.json:    "displaymanager",
displaymanager-20160506.json:    "displaymanager",
displaymanager-password-prompt-20160517.json:    "displaymanager-password-prompt"
displaymanager-password-prompt-gnome-SP1-20160519.json:    "displaymanager-password-prompt"
displaymanager-settings-20160322.json:    "displaymanager-settings"
displaymanager-settings-20160520.json:    "displaymanager-settings"
displaymanager-sles4sap-nousers-20160513.json:    "displaymanager",
first_boot-displaymanager-password-prompt-minimal-20160517.json:    "displaymanager-password-prompt"
gdm-workaround-bsc962806-20160125.json:    "displaymanager",
xdm-20150429.json:      "displaymanager",
xdm-20160504.json:    "displaymanager",
```